### PR TITLE
fix error about setting readonly variable

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

fix the error about setting readonly variable
    - make variables in install wrapper

see: 
https://travis-ci.org/PowerShell/PowerShell/jobs/364351571#L501

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/install-powershell.sh [hunk 1]
```diff
@@ -1,154 +1,155 @@
...
-            DIST="${DIST}[`cat /etc/UnitedLinux-release | tr "\n" ' ' | sed s/VERSION.*//`]"
+            if [ -f /etc/UnitedLinux-release ] ; then
+                DIST="${DIST}[`cat /etc/UnitedLinux-release | tr "\n" ' ' | sed s/VERSION.*//`]"
+            fi
+            OS=`lowercase $OS`
+            DistroBasedOn=`lowercase $DistroBasedOn`
+            local DIST
+            local DistroBasedOn
+            local PSUEDONAME
+            local REV
```
Comment (daxian-dbw) [inline tools/install-powershell.sh]: Shall we move the `local` variable declarations up beside `OS`, `Kernel` and `Mach`?

**Review by daxian-dbw — APPROVED** [review]
Left one comment.
@adityapatwardhan It's easier to review using codeflow.

**Review by adityapatwardhan — APPROVED** [review]
:shipit:

**Comment by adityapatwardhan** [comment]
@TravisEz13 Please have a look at the Travis CI failure.

**Comment by TravisEz13** [comment]
The latest change fixed the Travis CI failure

**Pull request merged**